lab-cameron/lib/sll.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
**TODO FOR LOST POINTS**

You need to comment what the Big Oh notation of your methods are.